monero/backends/jsonrpc.py

```diff
@@ -183,6 +186,91 @@ def balances(self, account=0):
 
+    def get_unspent_inputs(
+        self,
+        account_index: int = 0,
+        subaddr_indices: Optional[List[int]] = None,
+        verbose: bool = False
+    ) -> List[Input]:
+        """
+        Fetches the unspent inputs in the Wallet.
+
+        Args:
+            account_index: The index of the account whose Unspent Inputs we
+            want to fetch.
+            subaddr_indices: If it's provided, this function only fetches the
+            Unspent Inputs addressed to the Subaddress whose indices are
+            included in this list.
+            verbose: Whether to include key_image in the RPC response.
+        """
+        params = {
+            "transfer_type": "available",
+            'account_index': account_index,
+            'verbose': verbose
+        }
+        if subaddr_indices:
+            params['subaddr_indices'] = subaddr_indices
+
+        method = "incoming_transfers"
+        inputs = self.raw_request(method, params)["transfers"]
+
+        return [
+            Input(
+                atomic_amount=elem.get("amount"),
+                global_index=elem.get("global index"),
+                key_image=elem.get("key image"),
+                spent=elem.get("spent"),
+                subaddr_index=elem.get("subaddr index"),
+                tx_hash=elem.get("tx hash"),
+                tx_size=elem.get("tx size"),
+            )
+            for elem in inputs
+        ]
+
+    def get_incoming_transactions(
+        self,
+        account_index: int = 0,
+        unconfirmed: bool = False,
+        min_height: int = 0,
+        subaddr_indices: Optional[List[int]] = None
+    ) -> List[IncomingPayment]:
+        """
+        Calls the wallet RPC to get incoming transactions.
+
+        Args:
+            account_index: The index of the account we want to query.
+            unconfirmed: Whether to include unconfirmed transactions (the ones
+            in the pool).
+            min_height: The height of the block where we want to start scanning
+            from.
+            subaddr_indices: If this is included, this function only fetches
+            the the Incoming Transactions addressed to the Subaddresses whose
+            indices are included in this list
+        """
+        params: Dict[str, Any] = {
+            'account_index': account_index,
+            "in": True,
+            'out': False
+        }
+
+        if unconfirmed:
+            params["pool"] = True
+
+        if min_height:
+            params["filter_by_height"] = True
+            params["min_height"] = min_height
+        
+        if subaddr_indices:
+            params['subaddr_indices'] = subaddr_indices
+
+        resp = self.raw_request("get_transfers", params)
+        transfers = resp.get("in", [])
+        if unconfirmed:
+            transfers.extend(resp.get("pool", []))
+
+        return list(map(self._inpayment, transfers))
+
+    # TODO: consider removing/deprecating
```

Comment:
Why don't you create an issue describing why it should be deprecated?

Reply:
Good point. Created: https://github.com/Domol/monero-python/issues/10